File: ois/ois20/round2/ois11_round2_solutions/examroom.pas
{ input data }
var
    R, C, K  : longint;

begin
{
    uncomment the following lines if you want to read/write from files
    assign(input,  'input.txt');  reset(input);
    assign(output, 'output.txt'); rewrite(output);
}

    read(R);
    read(C);
    read(K);
    readln();

    writeln(((R + K - 1) div K) * ((C + K - 1) div K));
end.

File: ois/ois20/round2/ois11_round2_solutions/examroom.cpp
// insert brief description of the solution here

#include <assert.h>
#include <stdio.h>

// input data
long long R, C, K;

int main() {
  //  uncomment the following lines if you want to read/write from files
  //  freopen("input.txt", "r", stdin);
  //  freopen("output.txt", "w", stdout);

  assert(3 == scanf("%lld %lld %lld", &R, &C, &K));
  printf("%lld\n", ((R + K - 1) / K) * ((C + K - 1) / K));
  return 0;
}
